packages/contracts-bedrock/test/fixtures/test-deploy-superchain-out.toml

```diff
@@ -1,3 +1,16 @@
+[dsi]
+paused = false
+
```

Comment:
Potentially add comments in here.

Reply:
I'd say we can do this in a follow up PR post-MVP

Reply:
Resolving, I made a figjam ticket for this.